test/fixtures/shared_vpc/outputs.tf

```diff
@@ -97,4 +97,5 @@ output "forseti-client-service-account" {
 output "forseti-server-service-account" {
   description = "Forseti Server service account"
   value       = "${module.forseti-shared-vpc.forseti-server-service-account}"
-}
\ No newline at end of file
+}
```

Comment:
Avoid changing whitespace; it doesn't add value and adds noise to the history.

You can remove this with the following:

```
git checkout master test/fixtures/shared_vpc/outputs.tf
```

This will change `test/fixtures/shared_vpc/outputs.tf` to the version of this file that's on master.